apps/apps_container.cpp

```diff
@@ -241,6 +241,7 @@ bool AppsContainer::switchTo(App::Snapshot * snapshot) {
 }
 
 void AppsContainer::run() {
+  setSharedContainer(this);
```

Comment:
Remove this. Instead, fill s_sharedContainer when building AppsConainerStorage in AppsContainerStorage::sharedContainer().